src/_layouts/default.html

```diff
@@ -0,0 +1,50 @@
+<!doctype html>
+<html lang="en">
+  <head>
+      <meta charset="utf-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+      <title>Home</title>
+
+      <link href="/css/styles.css" rel="stylesheet">
+      <link href="http://fonts.googleapis.com/css?family=Lato&subset=latin,latin-ext' rel='stylesheet' type='text/css">
+  </head>
+
+  <body>
+    <div class="sidenav">
+      <div class="mainnav">
+        <a href="/home"> <img src="https://cdn.glitch.com/779b971d-9c2d-457f-ac8f-945d722901a7%2Fblacksauce.png?v=1605829269133"> </a>
+        <ul>
+            <li><a href="/about">about</a></li>
+            <li><a href="/order">order</a></li>
+            <li><a href="/biscuit">biscuit talk</a></li>
+            <li><a href="/contact">contact</a></li>
+        </ul>
+      </div>
+      <footer>
+        <div class="info">
+            <h4>Baltimore, MD</h4>
+            <h4>info@blacksaucekitchen.com</h4>
```

Comment:
You can use an `a` element here to make the email address clickable:
```suggestion
            <h4><a href="mailto:info@blacksaucekitchen.com">info@blacksaucekitchen.com</a></h4>
```